change/@fluentui-react-menu-516d0998-e682-438e-9b8f-5e111ee77d3c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Prettier formatting",
```

Comment:
nit:
```suggestion
  "comment": "chore: Prettier formatting",
```

Reply:
Might be better to leave the menu changes out. If you have trouble with restoring those files, you can checkout the files from master and then commit using the `--no-verify` flag so they don't change back.